modules/connatixBidAdapter.js

```diff
@@ -140,6 +158,52 @@ export const spec = {
 
     _handleEids(requestPayload, validBidRequests);
 
+    if (window.pbjs) {
+      window.pbjs.onEvent(EVENTS.AUCTION_TIMEOUT, (timeoutData) => {
+        const isConnatixTimeout = timeoutData.bidderRequests.some(bidderRequest => bidderRequest.bidderCode === BIDDER_CODE);
+
+        if (isConnatixTimeout) {
+          const timeout = timeoutData.timeout;
+          // eslint-disable-next-line no-console
+          console.log(timeout);
+
+          ajax('ENDPOINT_BASR_URL' + '/timeout-route-name', null, JSON.stringify({timeout}), {
+            method: 'POST',
+            withCredentials: false
+          });
+        }
+
+        // eslint-disable-next-line no-console
+        console.log('Connatix auction timeout', timeoutData);
+      });
+      window.pbjs.onEvent(EVENTS.AUCTION_END, (auctionEndData) => {
+        const bidsReceived = auctionEndData.bidsReceived;
+
+        const hasConnatixBid = bidsReceived.some(bid => bid.bidderCode === BIDDER_CODE);
+        const connatixBid = bidsReceived.filter(bid => bid.bidderCode === BIDDER_CODE);
```

Comment:
this should use find as filter would produce and array - but the following code acts as connatixBid is a single element

Reply:
fixed